core/src/main/resources/hudson/tools/ZipExtractionInstaller/help.html

```diff
@@ -1,6 +1,6 @@
 <div>
     Downloads a tool archive and installs it within Jenkins's working directory.
-    Example: <code>http://apache.promopeddler.com/ant/binaries/apache-ant-1.7.1-bin.zip</code>
+    Example: <code>https://downloads.apache.org/ant/binaries/apache-ant-1.10.12-bin.zip</code>
     (or whatever mirror is closest to your server)
```

Comment:
While the old URL was weird, now the mirror comment makes no sense anymore. Delete the part in parentheses and wipe all translations?